app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,26 @@
+class OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  def google_oauth2
+    @user = User.from_omniauth(request.env["omniauth.auth"])
+
+    if @user.persisted?
+      flash[:notice] = "devise.omniauth_callbacks.success", :kind => "Google"
```

Comment:
The problem is that you removed a reference to a module (`I18n`) and a method call (#t). The original code is passing two arguments to a method. The method returns a string based on the data in devise.en.yml and the arguments, and the string is then added to the flash hash with the key `:notice`. 

Once you remove those parts, you're effectively trying to assign two values to one hash key, which may cause a Ruby exception, and even if it doesn't, the resulting value in the hash will literally be "devise.omniauth_callbacks.success", instead of the nicely worded message stored in devise.en.yml under the keys specified by `devise.omniauth_callbacks.success`.

This module is typically used for translations, but in this particular case, we can just leverage it to reference one of the English messages that Devise has already provided for us. According to the guide, it's included in all Rails apps by default starting with Rails 2.2, and is actually used behind the scenes for a wide range of messages, such as Active Record validation errors.